test/unit/test_loss_image.py

```diff
@@ -55,6 +55,11 @@ def test_dissimilarity_fn():
     )
     assert is_equal_tf(get_zero_similarity_ssd, [0, 0])
 
+    #  testing if we can get [0, 0] (i.e. zero vector) in mutual information
+    t = tf.ones([4, 3, 3, 3, 1])
+    get_zero_similarity_gmi = image.dissimilarity_fn(t, t, "gmi")
+    assert tf.reduce_max(tf.abs(get_zero_similarity_gmi - [0, 0, 0, 0])).numpy() < 5e-6
```

Comment:
@QianyeYang why we can't use the `is_equal_tf` here?

Reply:
I actually would add that the use of consistent asserting functions is a good practice for future maintenance of the tests - for example, when the machine precision changes...

Reply:
@mathpluscode Because the EPS in that function is defined as 1e-6, but the value here is 2e-6.

Reply:
@QianyeYang This is a bit strange? as we are expecting zero values....
maybe for some division you need to add eps for both denominator and numerator

Reply:
> 
> 
> @QianyeYang This is a bit strange? as we are expecting zero values....
> maybe for some division you need to add eps for both denominator and numerator

will have a try adding eps on numerator, thanks!